api/src/main/java/org/openmrs/module/patientflags/api/FlagService.java

```diff
@@ -259,6 +342,15 @@
 	@Transactional(readOnly = true)
 	@Authorized(value = { "Manage Flags", "Test Flags" }, requireAll = false)
 	public DisplayPoint getDisplayPoint(Integer displayPointId);

```

Comment:
You need to have at least a line between methods.

Reply:
Done